src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -40,4 +40,36 @@ public void end(boolean interrupted) {}
   public boolean isFinished() {
     return false;
   }
-}
+
+  //take many botton's
+  private int movewristp() {
+    //input current locations, 
+
+    //could have 4 bottons 
+    //down, up, midle
+
+    //output instructions 
+    return 1;
+  }
+
+  private void wristfollow() {
+   //input:current position
+
+    // keeps the grip parallel to the arm
+
+    // output: instructions
+  }
+
+  public void getwristPostion() {
+    //gets the votage 
+  
+  } 
+
+  public double setwrsitpostion() {
```

Comment:
Suggestion: Change return type to void. This function will not be returning data.